Strategy11/formidable-forms
# Prevent Form Submission in WP Block Editor

This PR addresses an issue that allows forms to be submitted in the WP Block Editor, causing the page to reload. The proposed changes prevent this from happening.

## QA URL:
https://qa.formidableforms.com/sherv/wp-admin/post.php?post=1&action=edit

## Related Issue:
Fixes https://github.com/Strategy11/formidable-pro/issues/4246

## Testing Instructions:
1. Navigate to the block editor and add a Form using the Formidable Form block.
2. Try to submit the form on the block editor page.
3. Confirm that the form cannot be submitted and the page does not reload.
